globalconf_test.go

```diff
@@ -96,6 +96,27 @@ func TestParse_GlobalWithDottedFlagname(t *testing.T) {
 	}
 }
 
+func TestParse_GlobalWithEmptyValue(t *testing.T) {
+	t.Log("Given the need to test overwriting a flag with an environmental variable set to an empty string")
+	t.Log("\tReset the flags and clear the env")
+	{
+		resetForTesting("")
+	}
+
+	t.Log("\tSet the env var for the c flag to an empty string")
+	os.Setenv(envTestPrefix+"C", "")
+	t.Log("\tDefine the c flag and parse the INI file")
+	{
+		flagC := flag.String("c", "", "")
+
+		parse(t, "./testdata/global.ini", envTestPrefix)
+		t.Log("\t\tExpect flagC to an empty string and not be the value Hello World from the INI file")
+		if *flagC != "" {
+			t.Errorf("\t\tflagC found %v, expected an empty string", *flagC)
```

Comment:
This is nice. By the time I joined the Go projects I'm working on, the testify package was already adopted and I'm used to seeing `require.Equal(t, expected, got)`